### ISSUE TITLE
support newer PAGE-XML versions

### ISSUE THREAD

**Issue by bertsky** [mauvilsa/nw-page-editor#8]
AFAICS the latest version nw-page-editor can open/import is the `2017-07-15` schema (although the import transform does not seem to do anything really). When I open a 2018 or 2019 PAGE-XML, I get: `Expected as input a Page document`.

What would be required to get them to work? (I have spawned a new `xslt/page_from_2019-07-15.xslt` from `xslt/page_from_2017-07-15.xslt` and added this to `js/web-app.js` as well as `js/nw-app.js`, but still get the same error.)

Maybe the XSLT needs to actually remove (or rename/map) elements and attributes that have been added to the schema since 2013? See [here](https://github.com/PRImA-Research-Lab/PAGE-XML/blob/master/pagecontent/documentation/changelog.txt) for a summary of what has changed.

**Comment by mauvilsa** [comment]
@bertsky the 2017-07-15 import didn't do much. Its only purpose was to change the schema so that page2svg.xslt works. I had planed to improve the schema import support for a long time but had not yet implemented it. I just did and pushed the changes. Now all prima schema versions should import correctly, see [page2page.xslt#L13-L18](https://github.com/mauvilsa/nw-page-editor/blob/master/xslt/page2page.xslt#L13-L18). I also added a section to the readme explaining the support for formats [README.md#supported-formats](https://github.com/mauvilsa/nw-page-editor/blob/master/README.md#supported-formats).

I have not tested much, so I would appreciate if you test it with what you were doing. I also haven't checked if the changes broke something in the web server variant.
